libhb/common.h

```diff
@@ -475,7 +475,7 @@ typedef enum
  *****************************************************************************/
 struct hb_job_s
 {
-    PRIVATE const char  * json;   // JSON encoded job string
+    PRIVATE char  * json;   // JSON encoded job string
```

Comment:
Should actually do the opposite.  Remove PRIVATE instead of const.  These values are always const.  PRIVATE indicates the value is read only to users of libhb but writagle by libhb.

Reply:
OK, will change that.